test/test_automl/test_automl.py

```diff
@@ -781,39 +711,39 @@ def data_input_and_target_types():
     ys_binary = [
         y_binary_ndarray,
         list(y_binary_ndarray),
-        csr_matrix(y_binary_ndarray),
+        # csr_matrix(y_binary_ndarray),
```

Comment:
Why are these now commented out?

Reply:
I was going to say that we don't support sparse_y for now so this saves some test time but given the name of the test, I think it makes sense to reinclude it.

I'll fix this.

Reply:
Done

Reply:
There's one comment still added for multioutput regression. Is that on purpose?

Reply:
Nope, good spot, fixed.